README.rst

```diff
@@ -1,14 +1,22 @@
 Fake juju
```

Comment:
Nice idea. The rendering of your exact suggestion was a bit weird, so as a compromise I'm rendering the image *above* the Fake juju header. If you have ideas to improve that let me know (I tried to align the image to the right, but I wasn't able to).